CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.3.1] - 2020-07-25
```

Comment:
```suggestion
## [1.3.1] - TBD
```

This should not be a past date. I know you have deployed this in prod already but for tracking these changes nicely, we need to have this as the date when this module will be released here.